scripts/tools/zap/run_zaptool.sh

```diff
@@ -1,52 +0,0 @@
-#!/bin/bash
```

Comment:
What is the replacement for this?  People run ZAP to modify the .zap files in our tree.  What are they supposed to do in the new setup?

In particular, with this file removed but the developer documentation still saying to run it, we are not in a good place for people to be able to develop.

Reply:
added back with different logic (to account for zap devel or pre-install). I was not aware this script did more than 'run zap from third_party ... now I am.